src/lib.rs

```diff
@@ -123,7 +123,7 @@ pub fn try_init_custom_env(environment_variable_name: &str) -> Result<(), log::S
     let mut builder = formatted_builder();
 
     if let Ok(s) = ::std::env::var(environment_variable_name) {
-        builder.parse(&s);
+        builder.parse_filters(&s);
```

Comment:
Do we need to increase the minimum env_logger version? I think so, otherwise if the dependency tree just has 0.6.0, the method won't exist.

Reply:
Oh ok. Better now?